src/android/LocalyticsPlugin.java

```diff
@@ -1067,6 +1092,9 @@ public void run() {
         } else if (action.equals("setLocationListener")) {
             locationListener = new CDLocationListener(callbackContext);
             Localytics.setLocationListener(locationListener);
+            PluginResult result = new PluginResult(PluginResult.Status.OK);
+            result.setKeepCallback(true);
+            callbackContext.sendPluginResult(result);
             return true;
         } else if (action.equals("removeLocationListener")) {
             locationListener = null;
```

Comment:
Dont we need callbackContent.success() here to free up the listener that has been kept in removeLocationListener.

Reply:
nope